docs/faq.rst

```diff
@@ -65,27 +65,8 @@ I get import errors on libraries that depend on C modules
 
 This happens because our build system doesn't have the dependencies for building your project. This happens with things like libevent and mysql, and other python things that depend on C libraries. We can't support installing random C binaries on our system, so there is another way to fix these imports.
 
-You can mock out the imports for these modules in your ``conf.py`` with the following snippet::
+You can use this feature included in autodoc to mock http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports.
```

Comment:
Maybe we can put the name of the _feature_ (and the link to it here) and end with "..to mock your dependencies."

Reply:
Echoing proper format for linking instead of naked link